Trading/Mode/grid_trading_mode/grid_trading.py

```diff
@@ -440,7 +440,7 @@ async def _generate_staggered_orders(self, current_price, ignore_available_funds
                 highest_buy = current_price
                 lowest_sell = current_price
                 origin_created_buy_orders_count, origin_created_sell_orders_count = self._get_origin_orders_count(
-                    sorted_orders, recently_closed_trades
+                    recently_closed_trades, sorted_orders
```

Comment:
:open_mouth:

Reply:
it has not consequence but i prefer fixing it ^^